packages/block-editor/src/components/block-mobile-toolbar/test/block-actions-menu.native.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * External dependencies
```

Comment:
I realize we've been using that consistently, but aren't these actually internal dependencies since they're from inside the repo?

Reply:
I think it's because they're outside of the `packages` folder, the linting marks them as external dependencies.